util/util.go

```diff
@@ -0,0 +1 @@
+package util
```

Comment:
It's customary to include a `doc.go` briefly stating the purpose of the package and any sub packages.